docs/team/chewterence.md

```diff
@@ -12,30 +12,63 @@ regularly on the specified days and times every week.
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the Pool command feature. (Pull requests [\#121](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/121))
+* **Code contributed**: View the code that I have contributed at my
+  [RepoSense link](https://nus-cs2103-ay2021s2.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=&tabOpen=true&tabType=authorship&tabAuthor=chewterence&tabRepo=AY2021S2-CS2103T-W10-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code).
+
+* **New Feature**: Added the `pool` command feature. (Pull requests [\#121](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/121))
   * What it does: Allows the user to create a pool from the list of passengers, and add it to the list of pools in the GME system.
   * Justification: This feature is the core feature of the product, where pools are created and added to the pool list by the user.
+  * Highlights: Initially the command was implemented as the `drive` command which edits the list of passengers to include a driver with a phone
+    number. The model was eventually updated to include a list of pools, and the logic of the pool command was required to be changed,
+    this required an amount of refactoring of the code. Finally, towards the final iterations of the project, trip day and time validation was required to be
+    added into the pool command, which further enhances the product.
 
-* **New Feature**: Added Unpool command feature. (Pull requests [\#101](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/101), 
+* **New Feature**: Added the `unpool` command feature. (Pull requests [\#101](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/101), 
   [\#121](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/121))
   * What it does: Allows the user to remove a pool from the existing list of pools.
   * Justification: This feature complements the core feature of the product, as pools within the pool list could either 
     already be completed or entered with wrong details.
+  * Highlights: Initially this command was implemented as the `undrive` command that edits the assigned driver of a passenger to be empty. However, after
+    reconsidering and conceptualizing how the user would use the product, the model was updated to include a list of pools which the `unpool` command would then delete from.
+    Implementation of this command was the precursor that lead to the consideration of redesigning the model to include both a pool and passenger list.
```

Comment:
maybe _that led_ since the implementation of the command _**was** the precursor_